viewbuilder-android/src/main/java/com/squareup/viewbuilder/LayoutViewBuilder.kt

```diff
@@ -25,6 +25,33 @@ import com.squareup.coordinators.Coordinators
 import com.squareup.viewbuilder.ViewBuilder.Registry
 import io.reactivex.Observable
 
+/**
+ * Takes a [layoutId] and a [coordinatorConstructor] method and implements
+ * [ViewBuilder].
+ *
+ * Typical usage is to have a [Coordinator]'s `companion object` implement
```

Comment:
…delegating _to_ one of these…

Reply:
Also forgot to note that that the constructor can ask for the `ViewBuilder.Registry`.